app/models/concerns/organizer_position/has_role.rb

```diff
@@ -6,7 +6,18 @@ module HasRole
     included do
       # The enum values will allow us to have a hierarchy of roles in the future.
       # For example, managers have access to everything below them.
-      enum :role, { reader: 5, member: 25, manager: 100 }
+
+      enum :role, {
+        reader: 5,
+        member: 25,
+        manager: 100
+      }
+
+      roles.each do |role_name, role_value|
+        scope role_name, -> { where(role: role_name) }
```

Comment:
<img width="340" height="57" alt="Screenshot 2025-07-23 at 12 19 32 AM" src="https://github.com/user-attachments/assets/b52090d2-8bcf-4b51-8cca-574fd7900d71" />

I believe we get this for free with `enum`?